packages/client/tests/functional/methods/upsert/native-atomic/tests.ts

```diff
@@ -190,25 +190,25 @@ testMatrix.setupTestSuite(
     test('should only use ON CONFLICT when there is only 1 unique field in the where clause', async () => {
       const name = faker.person.firstName()
 
-      await expect(() =>
-        // This will fail
```

Comment:
By the way, is there a way to get this old behavior back?
Or is there no reason anyone would ever want that?

Reply:
Previously it errored with 
```
    Invalid `client.user.upsert()` invocation in
    .../upsert/native-atomic/tests.ts:195:21·
      192·
      193 await expect(() =>
      194   // This will fail
    → 195   client.user.upsert({
              where: {
                id: \"1\",
                name: \"Herta\"
              },
              ~~~~~~~~~~~~~~~
              create: {
                name: \"Herta\"
              },
              update: {
                name: \"Herta\"
              }
            })·
    Argument `where` of type UserWhereUniqueInput needs exactly one argument, but you provided id and name. Please choose one.
```
`UserWhereUniqueInput` can now take more than one since `extendedWhereUnique` is GA here.

So no you can't get the old behavior back, I don't think it makes sense, it didn't work before anyway.

Reply:
I meant the behavior of Prisma not allowing that. No idea if some user might actually prefer that...

Reply:
People are very happy with extended querying thanks to `extendedWhereUnique` so 🤷🏼, it does not make sense to me at the moment.